src/components/App.tsx

```diff
@@ -29,6 +30,20 @@ interface IAppProps extends RouteComponentProps<any> {
 @inject('mapStore', 'loginStore')
 @observer
 class App extends React.Component<IAppProps, IAppState> {
+    async componentWillMount() {
+        const pathName = navigator.getPathName();
+        if (!pathName.includes(SubSites.afterLogin)) {
```

Comment:
Use standard react-router matchPath() method instead of doing it manually. Current implementation only checks if the url contains the 'afterLogin' anywhere in the url. matchPath checks this more properly and less risk of missmatches.